tests/foreman/api/test_syncplan.py

```diff
@@ -167,9 +172,40 @@ def test_create_sync_plan_with_invalid_random_name(self, name):
         @Feature: SyncPlan
 
         """
+        with self.assertRaises(HTTPError):
+            entities.SyncPlan(name=name, organization=self.org).create()
 
+    @data(*invalid_values_list())
+    def test_create_sync_plan_with_invalid_interval_negative(self, interval):
+        """@Test: Create a sync plan with invalid interval specified.
 
-@stubbed()
+        @Assert: A sync plan can not be created with invalid interval specified
+
+        @Feature: SyncPlan
+
+        """
+        with self.assertRaises(HTTPError):
+            entities.SyncPlan(
+                interval=interval,
+                organization=self.org,
+            ).create()
+
+    def test_create_sync_plan_with_empty_interval_negative(self):
+        """@Test: Create a sync plan with no interval specified.
+
+        @Assert: A sync plan can not be created with no interval specified.
+
+        @Feature: SyncPlan
+
+        """
+        with self.assertRaises(HTTPError):
+            sync_plan = entities.SyncPlan(organization=self.org)
+            sync_plan.create_missing()
+            del sync_plan.interval
+            sync_plan.create(False)
```

Comment:
I think only this line of code needs to be in the `with self.assertRaises` context manager.